build.sbt

```diff
@@ -52,38 +65,64 @@ lazy val commonSettings = Seq(
   libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % "test",
 )
 
-lazy val root = project.in(file("."))
-  .aggregate(modules:_*)
-  .settings(commonSettings:_*)
+lazy val root = project
+  .in(file("."))
+  .settings(commonSettings)
+  .settings(crossScalaVersions := Seq())
```

Comment:
I don't understand

Reply:
Copied from the discipline build :shrug: